base.d.ts

```diff
@@ -169,6 +169,92 @@ export type ParseOptions = {
 	```
 	*/
 	readonly parseFragmentIdentifier?: boolean;
+
+	/**
+	Specify a pre-defined schema to be used when parsing values. The types specified will take precedence over global parameters such as: `parseNumber`, `parseBooleans`, and `arrayFormat`.
+
+	Use this feature to override the type for a value. This can be useful when the type is ambiguous such as a phone number (see example 1 and 2).
+
+	Types specified here will be used even when global parsing options such as `parseNumber`, and `arrayFormat` are not enabled (see example 3).
+
+	NOTE: array types (`string[]` and `number[]`) will not work if `arrayFormat` is set to `none`.
```

Comment:
It needs to describe what will happen. Like "it will have no effect" or "it will throw".